beep/tests/test_structure.py

```diff
@@ -243,7 +243,7 @@ def test_get_interpolated_diagnostic_cycles_maccor(self):
         diagnostic_cycles_interpolated = \
             cycler_run.get_interpolated_diagnostic_cycles(min_n_steps_diagnostic=3,
```

Comment:
Since the deprecated version of `get_interpolated_diagnostic_cycles()` was removed, we should discard this test as well?

Reply:
I think we can keep it, it should still work with the revisions I'd imagine?